src/System.IO.MemoryMappedFiles/tests/MemoryMappedViewAccessor.Tests.cs

```diff
@@ -56,7 +57,7 @@ public void InvalidArguments()
             }
         }
 
-        [Theory]
+        [ConditionalTheory]
```

Comment:
What does `ConditionalTheory` do? Is it necessary to have, if you might throw `SkipTestException`?

Reply:
yes. ConditionalTheory is extension handling SkipTestException

Reply:
I didn't even know you could use ConditionalTheory without parms and make it functional :smile:

Reply:
that was port of change I did few month back to support dynamic skip e.g. skip after test method is executed. That is useful for cases when you cannot determine it as static property - like curl vs Socket HTTP handler. In this case one could detect the ability and we could split the test to two. but I think this is simpler.